internal/tui/nomad/pages.go

```diff
@@ -427,11 +473,23 @@ func GetPageKeyHelp(
 
 	viewportKeyMap := viewport.GetKeyMap()
 	secondRow := []key.Binding{viewportKeyMap.Save, keymap.KeyMap.Wrap}
-	thirdRow := []key.Binding{viewportKeyMap.Down, viewportKeyMap.Up, viewportKeyMap.PageDown, viewportKeyMap.PageUp, viewportKeyMap.Bottom, viewportKeyMap.Top}
+
+	if currentPage == JobTasksPage {
```

Comment:
I think we'll want `AllTasksPage` to show this too - you can likely reuse `p.HasAdminMenu()` here